aggregator/options.go

```diff
@@ -62,6 +62,7 @@ var (
 	// are issues with the instances taking over the shards and as such we need to switch
 	// the traffic back to the previous owner of the shards immediately.
 	defaultBufferDurationAfterShardCutoff = time.Hour
+	defaultCloseTimeout                   = 5 * time.Second
```

Comment:
As mentioned above, let's use the `gracefulShutdownTimeout` for this purpose instead of having a second shutdown timeout.